PYME/experimental/_triangle_mesh.pyx

```diff
@@ -23,8 +23,8 @@ cdef extern from 'triangle_mesh_utils.h':
         
     cdef struct face_t:
         np.int32_t halfedge
-        float normal[VECTORSIZE]
```

Comment:
these definitions should match whatever is in triangle_mesh_utils.h (which uses `float`, not `float32_t`). There could be an argument for changing triangle_mesh_utils.h to use sized types for float, although a) I don't know know where and/or if sized floating point types are defined in c (and if this is consistent across compilers) and b) I don't think it's nearly critical - whilst `int` means different things across architectures, I'm pretty sure that float always implies 32 bit and double always implies 64 bit.

Reply:
I remember needing `float_32t` for `float` on Windows. I could be mis-remembering. Either way, this doesn't break anything, I did it to make sure we absolutely were not using the wrong data type. Can easily be reverted once we find whatever's got meshing behaving differently than it used to.